plugin.video.youtube/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.youtube" name="YouTube" version="5.1.17" provider-name="bromix">
+<addon id="plugin.video.youtube" name="YouTube" version="5.2.1" provider-name="h0d3nt3uf3l, jdf76, bromix">
```

Comment:
Kolifanes doesn’t maintain the repo, he pretty much just hosts it and lets us do what we are doing. Would you prefer me to go ahead and change the addon.xml to my repo now or can we do that in the next release? We started off everything off kolifanes, and then we were going to move it off kolifanes. Mainly jdf76/repository is the clean version, where kolifanes github will be tested version.

Jeff

> On May 17, 2016, at 9:12 AM, Razzeee <notifications@github.com> wrote:
> 
> In plugin.video.youtube/addon.xml <https://github.com/xbmc/repo-plugins/pull/480#discussion_r63518374>:
> 
> > @@ -1,5 +1,5 @@
> >  <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
> > -<addon id="plugin.video.youtube" name="YouTube" version="5.1.17" provider-name="bromix">
> > +<addon id="plugin.video.youtube" name="YouTube" version="5.2.1" provider-name="h0d3nt3uf3l, jdf76, bromix">
> A little bit confusing to me, as the source is in Kolifanes repo and he is not named here?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/xbmc/repo-plugins/pull/480/files/5ab7db1a24d3b6ff8e2f8a4d594407523dcf890a#r63518374>